help/en/Acknowledgements.shtml

```diff
@@ -619,6 +619,9 @@
         <li>Alain Le Marchand, <!-- GitHub: AlanUS -->
          who provided a number of decoder definitions and updates</li>
 
+        <li>Ralf Lang, <!-- ralflang -->
+        who contributed the Markin CdB connection code.
```

Comment:
I don't know if it's important or not, but `Marklin` spells `M&auml;rklin`, or `Märklin`.

If you don't think it matters, keep it as is.